osm_field/fields.py

```diff
@@ -200,12 +201,14 @@ def deconstruct(self):
 
     def formfield(self, **kwargs):
         """
-        :returns: A :class:`~django.forms.CharField` with a
-            :class:`~osm_field.forms.OSMWidget`.
+        :returns: A :class:`~osm_field.forms.OSMFormField` with a
+            :class:`~osm_field.widgets.OSMWidget`.
         """
-        widget = OSMWidget(lat_field=self.latitude_field_name,
+        widget = OSMWidget(
+            lat_field=self.latitude_field_name,
             lon_field=self.longitude_field_name)
```

Comment:
Can you add a tailing comma before the `)` and put the `)` on a new line, please.

```python
widget = OSMWidget(
    lat_field=...,
    lon_field=...,
)
```